MapboxNavigation/NavigationViewController.swift

```diff
@@ -455,12 +455,12 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         }
     }
     
-    func ETAIsStale() {
+    func updateETA() {
         tableViewController?.updateETA(routeProgress: routeController.routeProgress)
     }
     
     func resetETATimer() {
-       updateETAInfoTimer = Timer.scheduledTimer(timeInterval: 30, target: self, selector: #selector(ETAIsStale), userInfo: nil, repeats: true)
+       updateETATimer = Timer.scheduledTimer(timeInterval: 30, target: self, selector: #selector(updateETA), userInfo: nil, repeats: true)
```

Comment:
This timer will leak the whole class unless we invalidate it in `deinit` or `viewDidUnload()`.